pkg/kubectl/cmd/get.go

```diff
@@ -212,6 +215,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			FilenameParam(enforceNamespace, &options.FilenameOptions).
 			SelectorParam(selector).
 			ExportParam(export).
+			IncludeUninitialized(includeUninitialized).
```

Comment:
This is affecting WATCH verb. @caesarxuchao It's not clear to me what the behavior should be.